### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by AlanSoto31**
## Description
Landing page

## How has this been tested?

Please mark the tests that you ran to verify your changes. If difficult to test, consider providing instructions so reviewers can test.

- [x] Manual testing
- [ ] System tests
- [ ] Unit tests
- [ ] None

## Checklist

- [x] CI pipeline is passing
- [x] My code follows the conventions of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added seed data to the database (if applicable)

## Release tasks

Add any tasks that need to be done before/after the release of this feature.

## Screenshots/Loom

Screenshot 2024-08-13 at 15 29 13

### CODE REVIEW

**Review by EdgarB — COMMENTED** [review]
Overall the page looks good. I only noticed this couple of points:
1. Should we show this page when you are authenticated? And, Should we update the root page to be this one?
2. Width and height of purple circle on figma is 120px by 120px and on implementation is 128px by 128px
3. font size of "We’re cooking
something special" text on figma is 34px and on the implementation is 36px

File: app/controllers/landing_pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class LandingPagesController < ApplicationController
+  allow_unauthenticated_access only: [:show]
+
+  helper_method :show_header?
```
Comment (LuigiR0jas) [inline app/controllers/landing_pages_controller.rb:4]: What is this method for?

File: app/views/landing_pages/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="flex flex-col items-center flex-1 px-5 py-10 bg-purple-dark">
+  <%= inline_svg_tag("main_logo_with_date.svg", class: "w-full mt-14") %>
```
Comment (LuigiR0jas) [inline app/views/landing_pages/show.html.erb:2]: We could reuse the same header used for the login page

**Review by LuigiR0jas — COMMENTED** [review]
Looks good! The only thing we're missing is redirecting all traffic to this page based on whether a feature flag is enabled or not

File: app/controllers/landing_pages_controller.rb (hunk 1 above)
Comment (AlanSoto31) [reply app/controllers/landing_pages_controller.rb:4]: `show_header?` is used to avoid rendering the header that is normally rendered on the main layout.

**Review by LuigiR0jas — APPROVED** [review]

**Pull request merged**